website/docs/docs/dbt-cloud-apis/sl-graphql.md

```diff
@@ -593,7 +619,7 @@ mutation {
 }
 ```
 
-**Querying compile SQL with saved queries** 
+#### Querying compile SQL with saved queries
 
 This query includes the field `savedQuery` and generates the SQL based on a predefined [saved query](/docs/build/saved-queries),rather than dynamically building it from a list of metrics and groupings. You can use this for frequently used queries.
```

Comment:
Can we remove the `# new field` comment in this example and the one below? It's not so new anymore 🙂